react/components/EXPERIMENTAL_Table/DataTable/Rows.tsx

```diff
@@ -15,7 +17,9 @@ const Rows: FC<RowsProps> = ({ cellProps, rowProps }) => {
         }
       : {}
     const isSelected =
-      bulkState && bulkState.selectedRows.some(row => row.id === rowData.id)
+      bulkContext &&
+      bulkContext.bulkState &&
+      bulkContext.bulkState.selectedRows.some(row => row.id === rowData.id)
```

Comment:
No chance of `selectedRows` being `null` or `undefined`?

Reply:
Nope, since `bulkState` initializes `selectedRows` with an empty array.